flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/container/ImageBuildException.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.flink.tests.util.flink.container;
+
+/** Exception indicating errors when building Flink image. */
+public class ImageBuildException extends Exception {
+    public ImageBuildException() {}
+
+    public ImageBuildException(String message) {
+        super(message);
+    }
+
+    public ImageBuildException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public ImageBuildException(Throwable cause) {
+        super(cause);
+    }
```

Comment:
Do we need all these constructors?

Reply:
Maybe it makes sense to only pass the imageName as an argument and template the string inside of the `ImageBuildException`